### DISCUSSION TITLE
Hide line numbers

### DISCUSSION THREAD

**Discussion by hatamiarash7** [Textualize/rich#2231] [Q&A] [answered]
I have `file name` with `line` number in every line of the console. for example in this SC :

`a.py:36`

![sdfsdfs](https://user-images.githubusercontent.com/16325641/165483962-008c0ff7-8513-4141-bf5d-ac37453c73b0.png)

how can I remove them??? I can't find anything. It is for Python or rich?

**Comment by willmcgugan** [comment, marked as answer]
Are you using the logging handler?

**Reply by hatamiarash7** [reply]
No. It's my code :

```py
from rich.console import Console

console = Console()

for message in messages:
    with console.status("[bold green]Converting ...", spinner='aesthetic') as status:
        ...
        console.log(f"[red]Convert[/red] \t [blue]{timestamp}[/blue] \t [red]to[/red] \t [blue]{rfc3339_str}[/blue]")

```

**Reply by willmcgugan** [reply]
You can set log_path on Console to False.

**Reply by hatamiarash7** [reply]
Good! Thank you ... For other users, it's the solution:

```py
console = Console(log_path=False)
```
